src/db/migrations/20-add-consumable-views.ts

```diff
@@ -126,6 +129,10 @@ export const down = async (knex: Knex) => {
   for (const table of Object.values(Table)){
     await knex.raw(`drop view "${tableNameToViewName(table)}"`);
   }
+  
+  await knex.schema.alterTable(Table.ipfsPins, table => {
+    table.string('status')
```

Comment:
missing the drop?

Reply:
the drop is in the up. The down adds the column back